src/OrchardCore.Themes/TheBlogTheme/Recipes/blog.recipe.json

```diff
@@ -843,6 +897,19 @@
         }
       ]
     },
+    {
+      "name": "media",
+      "Files": [
+        {
+          "TargetPath": "home-bg.jpg",
+          "SourcePath": "../wwwroot/img/home-bg.jpg"
+        },
```

Comment:
I would have thought this would go in the `App_Data\media` folder, not in the `wwwroot` folder, otherwise how would `asset_url` find it?

Reply:
The assets are imported in AppData\Media\{currentTenant}.